shared/test/test_net_sim_api.rb

```diff
@@ -71,6 +71,26 @@ def test_create_read_update_delete
     assert read_records.first.nil?, 'Table was not empty'
   end
 
+  def test_read_multiple_tables
+    create_record({name: 'rec1_1'}, 'table1')
+    create_record({name: 'rec1_2'}, 'table1')
+    create_record({name: 'rec2_1'}, 'table2')
+    create_record({name: 'rec2_2'}, 'table2')
+    create_record({name: 'rec3_1'}, 'table3')
+
+    @net_sim_api.get "/v3/netsim/#{@shard_id}?t[]=table1&t[]=table2@2&t[]=table3@2"
+    assert_equal 200, @net_sim_api.last_response.status
```

Comment:
This route appears to also respond to `/v3/netsim/#{@shard_id}` without a query string.  Is that a valid request, and should we have a short test to pin that behavior?